File: ContactManagerWeb/HarrisContactWeb/HarrisContactWeb/Models/PersonalContact.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HarrisContactWeb.Models
{
    public class PersonalContact
    {// Attributes of the Personal Contact
        public int PersonalContactID { get; set; }//Primary Key of Personal Contact
        public string PersonalFname { get; set; }//String Personal Contact Firstname
        public string PersonsalLname { get; set; }//String Personal Contact Lastname 
        public string PersonalEmail { get; set; }//String Personal Contact Email
        public string HomeTel { get; set; }//String Personal Contact Home Telephone
        public string PersonalAddr1 { get; set; }//String Personal Contact Address 1
        public string PersonalAddr2 { get; set; }//String Personal Contact Address 2
        public string PersonalCity { get; set; }//String Personal Contact City 
        public string PersonalPostCode { get; set; }//String Personal Contact PostCode

    }
}

File: ContactManagerJava/HarrisContactManager/src/PersonalContact.java
//Personal Contact class inherits from Contact class
public class PersonalContact extends Contact {
//Encapsulation of attribute
	private String homeTel;//Personal Contact attribute
//inherited attributes
	public PersonalContact(String id, String firstName, String lastName, String email, String addr1, String addr2,
			String city, String postCode, String homeTel) {
		//attributes of super class
		super(id, firstName, lastName, email, addr1, addr2, city, postCode);
		this.homeTel = homeTel;
	}
//get method for HomeTel
	public String getHomeTel() {
		return homeTel;
	}
//set method for HomeTel
	public void setHomeTel(String homeTel) {
		this.homeTel = homeTel;
	}

//Polymorphism-overriding toString method inherited from the contact class to string 
	@Override
	public String toString() {
		return "PersonalContact [homeTel=" + homeTel + "]";
	}

}
